content/influxdb/v2.0/install.md

```diff
@@ -1,12 +1,13 @@
 ---
-title: Get started with InfluxDB
-description: Download, install, and setup InfluxDB, creating a default organization, user, and bucket.
+title: Install InfluxDB OSS
+description: Download and install InfluxDB.
 menu:
   influxdb_2_0:
-    name: Get started
+    name: Install 
 weight: 2
 influxdb/v2.0/tags: [get-started, install]
 aliases:
+  - /influxdb/v2.0/get-started/
   - /influxdb/v2.0/introduction/get-started/
   - /influxdb/v2.0/introduction/getting-started/
 ---
```

Comment:
Let's not make this change as part of this PR. At this point, I'm not comfortable simply renaming the OSS get-started doc. We need to have both an Install and a Get Started doc in OSS. Remove this change from the PR and we can do it in another.